deployment/ansible/roles/raster-foundry.base/defaults/main.yml

```diff
@@ -9,3 +9,5 @@ envdir_config:
   RF_STATSD_HOST: "{{ statsite_host }}"
   RF_CACHE_HOST: "{{ redis_host }}"
   RF_CACHE_PORT: "{{ redis_port }}"
+  RF_PROFILE: "{{ aws_profile }}"
```

Comment:
If we use `AWS_PROFILE` here, we may be able to get away from having to explicitly pass the `profile_name` keyword argument. Boto [says](http://boto.readthedocs.org/en/latest/boto_config_tut.html) it respects this environment variable.